delfin/drivers/hitachi/vsp/rest_handler.py

```diff
@@ -0,0 +1,200 @@
+# Copyright 2020 The SODA Authors.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+import threading
+
+import requests
+import six
+from oslo_log import log as logging
+
+from delfin import cryptor
+from delfin import exception
+from delfin.drivers.hitachi.vsp import consts
+from delfin.drivers.utils.rest_client import RestClient
+
+LOG = logging.getLogger(__name__)
+
+
+class RestHandler(RestClient):
+    SYSTEM_URL = '/ConfigurationManager/v1/objects/storages'
+    LOGOUT_URL = '/ConfigurationManager/v1/objects/sessions/'
+    COMM_URL = '/ConfigurationManager/v1/objects/storages/'
```

Comment:
Should we end the url with '/' . I guess it is better to append "/" where evr it is concatinated, for better readability

Reply:
changed